src/Http/Ownership/ApproveOwnershipRequestHandler.php

```diff
@@ -42,9 +56,24 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             new ApproveOwnership(new Uuid($ownershipId))
         );
 
+        $ownership = $this->ownershipSearchRepository->getById($ownershipId);
+
+        if (!$this->itemHasCreator($ownership->getItemId())) {
+            $this->commandBus->dispatch(
```

Comment:
I wonder about the synchronisation of this flow. The sending of the ownership mails works with an extra event listener and for that listener there is a real async worker to avoid blocking on sending the mails.

So I think firing the `ChangeOwner` could lead to a weird situation where the organizer already has the new creator and the fallback would not trigger. I think the fallback will only trigger on the request and the helpdesk gets the request email, but not sure who get the approval mail.

This also makes me wonder if we should have 2 pull requests because 2 rather independent changes are done.

Reply:
After discussing this during the UPW retro meeting we can ignore this.